src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.WinMD.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Reflection.Metadata
 {
-    partial class MetadataReader
+    public partial class MetadataReader
```

Comment:
If we're going to put 'public' here, we should also put sealed IMHO to matched the other partial declaration.